### PR TITLE
Add compliance tab and different compliance states

### PR DESCRIPTION
**Pull request by fofr**
- Add compliance tab
- Creates filters for a variety of appointment types

Includes designs for:
https://ms-design-history.herokuapp.com/breach-iteration/
https://ms-design-history.herokuapp.com/breach-design-sprint/

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
   port: '3000',
 
   // Enable or disable password protection on production
-  useAuth: 'true',
+  useAuth: 'false',
```
Comment (benilovj) [inline app/config.js:13]: Could this be driven off environment variables in Heroku?

File: app/data/cases.js [hunk 2]
```diff
@@ -469,23 +509,23 @@ module.exports = [
...
-        'session-end-time': '2:30pm',
-        'repeating': 'No, it’s a one-off session',
-        'session-counts-towards-rar': 'No',
-        'did-service-user-comply': 'No',
-        'non-compliance-reason': 'Attended - failed to comply',
-        'confirmed': true,
-        'session-notes': `I called Dylan to confirm he had understood where he needed to be and when for his induction appointment. He was rude and abusive and in general very hostile in reaction to his sentence saying this wasn’t his fault. He mentioned he “doesn’t have time for this”. I reiterated that he must be at the office on Thursday as part of his sentence requirements and if he doesn’t it’s going to reflect very poorly.`,
-        'sessionId': 978
       }
+      // {
```
Comment (benilovj) [inline app/data/cases.js:513]: Do we want to be leaving this in?

File: app/data/cases.js [hunk 3]
```diff
@@ -1034,7 +1064,7 @@ module.exports = [
   },
   // Gary
   {
-    'hidden': true,
+    'hidden': false,
```
Comment (benilovj) [inline app/data/cases.js:1067]: Might as well get rid of this line.

File: app/data/cases.js (hunk 2 above)
Comment (fofr) [reply app/data/cases.js:513]: I've left in as an example of a phone appointment that can also be a FTC, but we didn't want it included in the scenarios for current research.

File: app/data/cases.js (hunk 3 above)
Comment (fofr) [reply app/data/cases.js:1067]: Also left this in as an example of how to hide a user. Otherwise we could remove the entire `hidden` feature.

Comment (benilovj) [reply app/data/cases.js:1067]: 👍

**Review by benilovj — APPROVED** [review]

**Pull request merged**